marlowe-dashboard-client/src/Play/View.purs

```diff
@@ -169,22 +170,42 @@ renderCard wallets newWalletDetails templates playState card =
                 Nothing -> []
       ]
 
-renderScreen :: forall p. WalletLibrary -> Screen -> State -> HTML p Action
-renderScreen wallets screen playState =
+renderScreen :: forall p. WebSocketStatus -> WalletLibrary -> Screen -> State -> HTML p Action
+renderScreen webSocketStatus wallets screen playState =
   let
     currentSlot = view _currentSlot playState
 
+    slotsInSync = view _slotsInSync playState
+
     templateState = view _templateState playState
 
     contractsState = view _contractsState playState
   in
     div
       -- TODO: Revisit the overflow-auto... I think the scrolling should be set at the screen level and not here.
       --       this should have overflow-hidden to avoid the main div to occupy more space than available.
-      [ classNames [ "absolute", "top-0", "bottom-0", "left-0", "right-0", "overflow-auto", "z-0" ] ] case screen of
-      ContractsScreen -> [ ContractHomeAction <$> contractsScreen currentSlot contractsState ]
-      WalletLibraryScreen -> [ walletLibraryScreen wallets ]
-      TemplateScreen -> [ TemplateAction <$> contractSetupScreen wallets currentSlot templateState ]
+      [ classNames [ "absolute", "top-0", "bottom-0", "left-0", "right-0", "overflow-auto", "z-0" ] ]
```

Comment:
setting all top, bottom, properties is the same as using `inset` 

https://tailwindcss.com/docs/top-right-bottom-left